library/systemd/README.md

```diff
@@ -0,0 +1,76 @@
+# Systemd actions
+
+The following table describes the actions to perform over a systemd service and its associated socked (if any) when we try to start, stop, restart or reload the service.
+
+
+| Status | Start | Stop | Restart | Reload |
```

Comment:
Note that the Restart column could be rephrased and unified, to always perform "stop, then start" (where "start" operates in the mode "nothing is running").
This would fix a bug in the current table: if both service+socket are running, and you have Manual start mode, and you Restart, it leaves the socket running.